synapse/tools/docker_validate.py

```diff
@@ -0,0 +1,158 @@
+import os
```

Comment:
Would we be better off making synapse/tools/docker/validate.py ? We can discuss :+1:

Reply:
That would be a fine change, especially if we ever offer any other tools